Shared/Samples/Show viewshed from geoelement in scene/README.md

```diff
@@ -0,0 +1,47 @@
+# Show viewshed from geoelement in scene
+
+Analyze the viewshed for an object (GeoElement) in a scene.
+
+![Image of show viewshed from geoelement in scene](show-viewshed-from-geoelement-in-scene.png)
+
+## Use case
+
+A viewshed analysis is a type of visual analysis you can perform on a scene. The viewshed aims to answer the question 'What can I see from a given location?'. The output is an overlay with two different colors - one representing the visible areas (green) and the other representing the obstructed areas (red).
+
+## How to use the sample
+
+Tap to set a destination for the vehicle (a GeoElement). The vehicle will 'drive' towards the tapped location. The viewshed analysis will update as the vehicle moves.
+
+## How it works
+
+1. Create and show the scene, with an elevation source and a buildings layer.
+2. Add a model (the `GeoElement`) to represent the observer (in this case, a tank).
+    * Use a `SimpleRenderer` which has a heading expression set in the `GraphicsOverlay`. This way you can relate the viewshed's heading to the `GeoElement` object's heading.
+3. Create a `GeoElementViewshed` with configuration for the viewshed analysis.
+4. Add the viewshed to an `AnalysisOverlay` and add the overlay to the scene.
+5. Configure the SceneView `CameraController` to orbit the vehicle.
+
+## Relevant API
+
+* AnalysisOverlay
+* GeodeticDistanceResult
+* GeoElementViewshed
+* GeometryEngine.DistanceGeodetic (used to animate the vehicle)
+* ModelSceneSymbol
+* OrbitGeoElementCameraController
```

Comment:
I can't remember, should these use code voice? Also, `GeometryEngine.DistanceGeodetic` isn't a symbol in the Swift API.

```suggestion
* `AnalysisOverlay`
* `GeodeticDistanceResult`
* `GeoElementViewshed`
* `static GeometryEngine.geodeticDistance(from:to:distanceUnit:azimuthUnit:curveType:)` (used to animate the vehicle)
* `ModelSceneSymbol`
* `OrbitGeoElementCameraController`
```

Reply:
No they don't need the backticks. Qualified symbols should be used